parts/openshift/release-3.9/openshiftmasterscript.sh

```diff
@@ -47,7 +47,7 @@ sed -i -e "s#--loglevel=2#--loglevel=4#" /etc/sysconfig/${SERVICE_TYPE}-master-c
 
 rm -rf /etc/etcd/* /etc/origin/master/* /etc/origin/node/*
 
-MASTER_OREG_URL="$IMAGE_PREFIX/$IMAGE_TYPE"
+MASTER_OREG_URL="$IMAGE_PREFIX/$IMAGE_TYPE-${component}:${version}"
```

Comment:
@kargakis is it correct that `${component}:${version}` is not shell /single/ quoted?  Can this work as is?

Reply:
dollar signs need to be escaped, fixed!